### ISSUE TITLE
`<c-w>gd` stopped working, now it behaves like Go To Definition instead of opening in a split to the side.

### ISSUE THREAD

**Issue by trusktr** [vscode-neovim/vscode-neovim#481]
It was working great, then all of a sudden it doesn't. This is what currently happens when I press `<c-w>gd` then select the thing in the peek:

https://user-images.githubusercontent.com/297678/103332122-bcfaeb80-4a1d-11eb-89d4-a00d0795b70e.mp4

**Comment by trusktr** [comment]
I can't get it to work anymore. It worked great a few times when I first tried it after you mentioned it in https://github.com/asvetliakov/vscode-neovim/issues/477#issuecomment-752330982, but that was it. Now it always goes to definition.

**Comment by asvetliakov** [comment]
Did you change any conf or have you added any keybinding while testing?

adding something like `ctrl+w 1` to vscode keybindings will stop sending `<C-w>` to neovim because vscode would start listening for second chord. in this case you need to configure `<C-w> 1` in neovim and call vscode

**Comment by trusktr** [comment]
> Did you change any conf or have you added any keybinding while testing?

Nope, just tried some of the things you mentioned (like search jump), then eventually I circled back to trying <c-w>gd and it didn't work anymore.

I tried restarting VS Code, but that didn't fix it. Looks like something got stuck for some reason. My vimrc and VS Code settings are the same, haven't modified them since earlier today, before all the screen caps I posted in the other thread.
